hokusai/production.yml

```diff
@@ -29,8 +29,10 @@ spec:
         - containerPort: 3000
         resources:
           requests:
-            cpu: "0.7"
+            cpu: "0.5"
+            memory: "512Mi"
```

Comment:
Would you mind describing what this section is for? I'm just curious!

Reply:
Sure!  Whenever a new pod is requested (whether manually or via autoscaling) the k8s scheduler tries to place the pod on a machine with available resources.  This `resources.requests` section describes the minimum cpu / memory that this pod should be allocated when placing it.  Tuning these values to [what the app consumes under a normal workload](https://kubernetes.artsy.net/#!/search?q=metaphysics&namespace=default) helps the scheduler place pods accurately.  More details [here](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container).

Reply:
Ah, cool, so it's a minimum! Cooool